Update README.md

laith43d opened the pull request:
- added section "embedded editors"
- added the embedded editor "TinyMCE"

laith43d committed:
Update README.md

- added section "embedded editors"
- added the embedded editor "TinyMCE"

Alkass commented:
Can you move the Embedded Editors section under [Editors](https://github.com/IQDevs/Awesome-IQDevs#editors)? Look at how the [Web Development](https://github.com/IQDevs/Awesome-IQDevs#web-development) section is done and replicate the structure

laith43d committed:
Update README.md

Added subsection to editors "Standalone editors"
Moved the subsection "embedded editors" one deeper level

Alkass commented:
Merged. Thanks
